.github/workflows/e2e.yml

```
@@ -0,0 +1,37 @@
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [22.x]
+
+    steps:
+      - name: Set Timezone
+        run: sudo timedatectl set-timezone America/Sao_Paulo
+
+      - name: Checkout Code
+        uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)